pixels.go

```diff
@@ -0,0 +1,98 @@
+package waifu2x
+
+import (
+	"fmt"
+	"math"
+)
+
+type Pixels struct {
+	Width  int
+	Height int
+	Pix    []uint8 // 0以下を0, 255以上を255 とする
+}
+
+func NewPixels(width, height int) *Pixels {
+	return &Pixels{
+		Width:  width,
+		Height: height,
+		Pix:    make([]uint8, width*height),
+	}
+}
+
+func (p Pixels) NewImagePlane() *ImagePlane {
+	pl := NewImagePlane(p.Width, p.Height)
+	for i := 0; i < len(p.Pix); i++ {
+		pl.Pix[i] = float64(p.Pix[i]) / 255.0
+	}
+	return pl
+}
+
+func (p Pixels) Decompose() (r, g, b, a *Pixels, err error) {
```

Comment:
__[golint]__
* [golint] exported method Pixels.Decompose should have comment or be unexported